content/07.pathogenesis.md

```diff
@@ -75,7 +75,7 @@ Cell entry proceeds in three steps: binding, cleavage, and fusion.
 First, the viral spike protein binds to a host cell via a recognized receptor.
 Coronaviruses can bind to a range of host receptors [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009], with binding conserved only at the genus level [@doi:10.1146/annurev-virology-110615-042301].
 Viruses in the betacoronavirus genus, to which SARS-CoV-2 belongs, are known to bind to the CEACAM1 protein, 5-N-acetyl-9-O-acetyl neuraminic acid (Neu 5,9 Ac2), and to the angiotensin-converting enzyme 2 (ACE2) [@doi:10.3390/v4061011].
-SARS-CoV-2 has a high affinity for the human ACE2 receptor, which is expressed in the vascular epithelium, other epithelial cells, and cardiovascular and renal tissues [@doi:10.1007/s00134-020-05985-9; @doi:10.1007/978-0-387-33012-9_85].
+SARS-CoV-2 has a high affinity for the human ACE2 receptor, which is expressed in the vascular epithelium, other epithelial cells, and cardiovascular and renal tissues [@doi:10.1007/s00134-020-05985-9; @doi:10.1007/978-0-387-33012-9_85], as well as many others [@doi:10.15252/msb.20209610].
```

Comment:
```suggestion
SARS-CoV-2 has a high affinity for human ACE2, which is expressed in the vascular epithelium, other epithelial cells, and cardiovascular and renal tissues [@doi:10.1007/s00134-020-05985-9; @doi:10.1007/978-0-387-33012-9_85], as well as many others [@doi:10.15252/msb.20209610].
```

Reply:
pretty sure this gets fixed in the other PR too, but didn't want to have it revert since the same line is changed. I think we should merge that one first, then this, and just check to make sure it works out.